index.bs

```diff
@@ -244,11 +244,14 @@ Additionally, if a file contains [=AV1 image sequences=] and the brand <code>[=a
 
 NOTE: As defined in [[!MIAF]], a file that is primarily an image sequence still has at least an image item. Hence, it can also declare brands for signaling the image item.
 
-  <h2 id="file-constraints">File constraints</h3>
+  <h2 id="file-constraints">General constraints</h3>
 
   The following constraints are common to files compliant with this specification:
   - The file shall be compliant with the [[!MIAF]] specification and list 'miaf' in the [=compatible_brands=] field of the [=FileTypeBox=].
   - The file shall list <code>'[=avif=]'</code> or <code>'[=avis=]'</code> in the [=compatible_brands=] field of the [=FileTypeBox=].
+  - If transformative properties are used in derivation chains (as defined in [[MIAF]]), they shall only be associated with items that are not referenced by another derived item. For example, if a file contains a grid item and its referenced coded image items, cropping, mirroring or rotation transformations are only permitted on the grid item itself.
```

Comment:
The only use-case I can think of that is even remotely plausible is if you have a HW encoder that only supports a certain minimum frame size (say 512x512). If you then want to create a 'grid' with tiles that are 256x256 this might now not be possible since there is no way for you to crop the input tiles.

On the other hand, I know HEVC supports to handle stuff like this, so I'm guessing AV1 does as well.

Reply:
I agree with the use case. Some times encoders are limited to produce multiples of power of 2, so cropping in grids could be needed...

Reply:
Per meeting, this should not be an issue since AV1 supports cropping if needed.